doc/source/index.rst

```diff
@@ -20,60 +20,16 @@ alias is `npf`.  For example,
 >>> npf.irr([-250000, 100000, 150000, 200000, 250000, 300000])
 0.5672303344358536
 
-
-Functions
----------
-
-.. currentmodule:: numpy_financial
-
-.. toctree::
-   :maxdepth: 4
-
-.. autosummary::
-
-   fv
-   ipmt
-   irr
-   mirr
-   nper
-   npv
-   pmt
-   ppmt
-   pv
-   rate
```

Comment:
seemed redundant to me to have this on the homepage and also on the main "documentation" page